src/H5TS.c

```diff
@@ -405,7 +405,8 @@ H5TS__mutex_acquire(H5TS_mutex_t *mutex, unsigned int lock_count, hbool_t *acqui
  *--------------------------------------------------------------------------
  */
 herr_t
-H5TSmutex_acquire(unsigned int lock_count, hbool_t *acquired){
+H5TSmutex_acquire(unsigned int lock_count, hbool_t *acquired)
+{
     FUNC_ENTER_API_NAMECHECK_ONLY
```

Comment:
I really hate the formatter.

Reply:
My fault for not using the correct version.